### PR TITLE
SR1.1.1: add standard deviation to data model and include a graph visualisation view

### PR DESCRIPTION
**Pull request by ailopezn**
Extend the graphs to show the daily standard dev.

### CODE REVIEW

File: inflammation/models.py [hunk 1]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```
Comment (ShekharNarayanan) [inline inflammation/models.py]:
```suggestion
def standard_deviation(data):
```
The function could replaced by ```np.std()```, adds to the readability of the code! If you think you need the whole function for some reason, consider changing its name to something more understandable! (for e.g. ```standard_deviation()```

File: inflammation/models.py [hunk 2]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```
Comment (ShekharNarayanan) [inline inflammation/models.py]:
```suggestion
    mean_of_data = np.mean(data, axis=0)
```
Replace ```mmm``` with something like ```mean_of_data``` with a more lucid variable to improve readability!

File: inflammation/models.py [hunk 3]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
+    return {'standard deviation': s_dev2}
```
Comment (ShekharNarayanan) [inline inflammation/models.py]: Return the output ```s_dev2``` instead of the dictionary output. Makes the output of the function more readily usable!

File: inflammation/models.py [hunk 4]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
```
Comment (ShekharNarayanan) [inline inflammation/models.py]: Update documentation here: if you agree with my suggestion of returning the ```s_dev2``` directly, the docstring can stay the same. If you think a dictionary object is more appropriate, change the documentation to say the exact type of output returned from the data!

**Review by ShekharNarayanan — CHANGES_REQUESTED** [review]

File: tests/test_models.py [hunk 5]
```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
+    ([1.0, 1.0, 1.0], 0),
+    ([0.0, 2.0], 1.0)
+])
```
Comment (ShekharNarayanan) [inline tests/test_models.py:42]: Include the following test cases/edge cases for a better prepared test function!
Standard deviation for one patient with multiple observations.
Standard deviation for two patients.
Graph includes a standard deviation graph.
Standard deviation function should raise an error if given empty data.
Computing standard deviation where deviation is different from variance.
Standard deviation function should give correct result given negative inputs.
Function should work with numpy arrays.

**Review by ShekharNarayanan — CHANGES_REQUESTED** [review]
